### ISSUE TITLE
htgettoken crashes if --nobearertoken is given and credkey is not found or specified

### ISSUE THREAD

**Issue by duncanmmacleod** [fermitools/htgettoken#73]
If `htgettoken` is executed with the `--nobearertoken` option and a `--credkey` is not given or found in `~/.config` then it crashes:

```pytb
$ htgettoken --nobearertoken
Traceback (most recent call last):
  File "/home/duncan/opt/mambaforge/envs/py311/bin/htgettoken", line 1488, in <module>
    main()
  File "/home/duncan/opt/mambaforge/envs/py311/bin/htgettoken", line 936, in main
    fullsecretpath = secretpath.replace("%credkey", credkey)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: replace() argument 2 must be str, not None
```

It looks like this was introduced by 48fa2562.

**Comment by DrDaveD** [comment]
Ok, I can reproduce this if I give a `-a` and `-i` option.  I'm guessing you have those set in $HTGETTOKENOPTS.

**Comment by duncanmmacleod** [comment]
This was reported originally in https://git.ligo.org/computing/helpdesk/-/issues/3994 with HTCondor responsible for the `htgettoken` invocation, see:

https://github.com/htcondor/htcondor/blob/723ced6b5c13178c3b731f362dac864b53cdcfbb/src/condor_credd/condor_credmon_oauth/condor_vault_storer#L189

**Comment by DrDaveD** [comment]
It was because there issuer wasn't passed to it, so it didn't find a credkey under the "default" issuer.  Fixed in #76 and htgettoken-1.18.
